CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+# Contributing to nxwoker
+
+🙏 We would ❤️ for you to contribute to `NxWorker` and help make it even better than it is today!
+
+# Developing
+
+Start by installing all dependencies:
+
+```bash
+pnpm
+```
+
+Run the tests:
+
+```bash
+pnpm test
+pnpm e2e
+```
+
+Run the playground app:
+
+```bash
+pnpm start
```

Comment:
There's no default project set right now, so the `start` script doesn't work.

Reply:
Pass the booking desktop app project name to the nx serve command in the start script and we're good to go. You're also welcome to set the angular project as the default project since that's the one we're going to be adding most generators for.